pl_bolts/models/gans/dcgan/components.py

```diff
@@ -0,0 +1,93 @@
+# Based on https://github.com/pytorch/examples/blob/master/dcgan/main.py
+import torch
+from torch import nn
+
```

Comment:
For components`. I think that we can club these, #427.
We could probably have something like `components.gan,DCGANGenerator` and `components.gan,DCGANDiscriminator`

Reply:
Yeah, let's discuss that in the linked issue.